### DISCUSSION TITLE
Adding Cassandra Store capabilities

### DISCUSSION THREAD

**Discussion by vamsee91** [feast-dev/feast#4261] [Q&A]
I see that there is a v0.3 branch with Cassandra store changes for both ingestion and serving, is there any plans on merging them on the master branch ?

**Comment by tokoko** [comment]
There is a [cassandra](https://docs.feast.dev/reference/online-stores/cassandra) online store in the latest version. Just note that it's a contrib, meaning it's not being tested in ci.
